Fix Torpedo

tmsvspn opened the pull request:
This fixes 3 errors in the TorpedoStore

tmsvspn committed:
Fix typo in fire

tmsvspn committed:
Extract Random generator

tmsvspn committed:
Fix Exception not thrown in fire

timiant commented on src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:
```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
+
```
SonarCloud vizsgálata alapján az a tanácsolom használj java.security.SecureRandom - ot

tmsvspn replied:
Ez jó ötlet, implementáltam.

tmsvspn committed:
Replace Random with SecureRandom in TorpedoStore

This is for security reasons, as requested by timiant
in the recent pull request
